BE/src/main/java/team07/airbnb/entity/AccommodationEntity.java

```diff
@@ -86,8 +86,9 @@ public void addPicture(String url) {
         pictures.add(new Pictures(this, url));
     }
 
-    public void addProduct(LocalDate date, int price) {
-        products.add(ProductEntity.ofOpen(this, date, price));
+    public void addProduct(LocalDate date, Integer price) {
+        if(price != null) products.add(ProductEntity.ofOpen(this, date, price));
+        else addProduct(date);
```

Comment:
```suggestion
    public void addProduct(LocalDate date, Integer price) {
        if(price == null) {
            addProduct(date);
            return;
        } 
        products.add(ProductEntity.ofOpen(this, date, price));
        
```

요건 스타일인데요 
1) 조건문에 부정식보다 긍정식 (not null 인지 검사 하는것보다 is null 인지 검사하는것)이 조금 더 읽기 편해지는거같아요!
2) 분기처리는 early return 을 사용해도 좋을꺼같습니다!
3) 제 스타일인데요!  if-else 문에는  `{ }` 괄호를 붙여주는 편이에요 c언어에서 나오긴 하지만 Dangling else 같은 잠재적 위험도 있고 해서요! 

한번 읽어봐주시면 좋을꺼같습니다! https://en.wikipedia.org/wiki/Dangling_else